.gitattributes

```
@@ -1 +1,2 @@
 test/data/filters/pdf/file-PdfFilter-gold.txt text eol=lf
+test/data/tmx/TMXComplianceKit/*.rtf binary
```

There is known configuration for binary text data.

```suggestion
*.RTF diff=astextplain
*.rtf diff=astextplain
*.odt diff=astextplain
```